packages/atlas/src/components/_video/VideoPlayer/VideoPlayer.styles.ts

```diff
@@ -395,6 +399,31 @@ export const Container = styled.div<ContainerProps>`
     }
   }
 
+  .vjs-text-track-display {
+    > div {
+      margin: 0 !important;
+
+      > div {
+        inset: unset !important;
+        bottom: 0.36em !important;
+        transition: bottom 150ms linear;
```

Comment:
I think we need to adjust this, linear easing doesn't look very nice. Best to ask Adam (since Kuba is away for 2 weeks starting today) to take a look and give feedback